agent/fw_hooks.h

```diff
@@ -13,7 +13,7 @@
 #include "php_execute.h"
 
 extern void nr_cakephp_enable_1(TSRMLS_D);
```

Comment:
Do we want to remove this old code because it is seemingly no longer called?

Reply:
Also `nr_cakephp_special_1` and `nr_cakephp_special_1` don't seem to be used any more.

Reply:
Good point, I have removed all of the old unused code for CakePHP: [d0a6282](https://github.com/newrelic/newrelic-php-agent/pull/983/commits/d0a6282fda9ccd2ef8ae002e7c49eeef4e042155)